File: NetBeansProjects/java/simulator/src/main/java/com/maicol/simulator/Option.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.maicol.simulator;

/**
 *
 * @author Maicol
 */
public class Option extends Force {

    private final String name;
    private LawOne lawOne;
    private LawTwo lawTwo;
    private LawThree lawThree;

    public Option(String name) {
        super(123.65f, 317.5f);
        this.name = name;
        this.lawOne = new LawOne();
        this.lawTwo = new LawTwo();
        this.lawThree = new LawThree();
    }

    public void review() {
        switch (name) {
            case "Law One" -> {
                System.out.println("Results with Law One: ");
                lawOne.optionReviewLawOne(name, this);
            }
            case "Law Two" -> {
                System.out.println("Results with Law Two: ");
                lawTwo.optionReviewLawTwo(name, this);
            }
            case "Law Three" -> {
                System.out.println("Results with Law Three: ");
                lawThree.optionReviewLawThree(name, this);
            }
            default -> System.out.println("This law doesn't exist");
        }
    }
}

File: php/exercises/3.simulator/php/option.php
<?php
include('force.php');
include("actionAndReaction.php");
include("inertia.php");
include("forceAndAcceleration.php");

class option extends force {

    use lawOne;
    use lawTwo;
    use lawThree;

    private string $name;

    public function __construct(string $name) {
        parent::__construct(123.65, 317.5);
        $this->name = $name;
    }

    public function review() {

        if ($this->name=="Law One"){
            echo "Results with Law One: <br>";
            $this->optionReviewLawOne();
        }else if($this->name=="Law Two"){
            echo "<br>Results with Law Two: <br>";
            $this->optionReviewLawTwo();
        }else if($this->name=="law Three"){
            echo "<br>Results with Law Three: <br>";
            $this->optionReviewLawThree();
        } else {echo "this law don't exists";}
     }
}
?>